lib/moonshot/build_mechanism/travis_deploy.rb

```diff
@@ -79,9 +85,24 @@ def wait_for_build(version)
     end
 
     def wait_for_job(job_number)
-      cmd = "bundle exec travis logs #{@cli_args} #{job_number}"
-      # This log tailing fails at the end of the file. travis bug.
-      sh_step(cmd, fail: false)
+      # Find the travis job.
```

Comment:
I had considered this, but left it here since it's the only place it's currently used. I could move it to a separate method and only call it here. We could move it to an earlier point if we refactor the other methods to use the client rather than the CLI (which I feel we should do long-term, but is out of the scope of this story).